telethon/events/callbackquery.py

```diff
@@ -26,9 +26,13 @@ class CallbackQuery(EventBuilder):
             can use ``re.compile(b'data_')``.
     """
     def __init__(
-            self, chats=None, *, blacklist_chats=False, func=None, data=None):
+            self, chats=None, *, blacklist_chats=False, func=None, data=None, pattern=None):
         super().__init__(chats, blacklist_chats=blacklist_chats, func=func)
 
+        if data is None and pattern is None:
```

Comment:
With this code:
```@client.on(events.CallbackQuery)
async def handler(event):
    ...```

The condition will be `True` and a warning logged. This is probably not what we want.

Reply:
my bad I wanted it to work if he passed both.